content/docs/addons.md

```diff
@@ -4,46 +4,46 @@ title: Add-Ons
 permalink: docs/addons.html
 ---
 
-> Note:
+> Not:
 >
-> `React.addons` entry point is deprecated as of React v15.5. The add-ons have moved to separate modules, and some of them have been deprecated.
+> `React.addons` giriş noktası React v15.5'ten itibaren kullanımdan kaldırılmıştır.. Bu eklentiler ayrı modüllere taşındı ve bazıları kullanımdan kaldırıldı.
```

Comment:
not 2 periods :

> `.` instead of `..`

Reply:
Weird! I can't see in this my code! @alioguzhan